cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/ProgramNotificationSubscriberService.java

```diff
@@ -772,6 +774,13 @@ private void emitRunTimeMetric(ProgramRunId programRunId, ProgramRunStatus progr
     metricsContext.gauge(Constants.Metrics.Program.RUN_TIME_SECONDS, runTime);
   }
 
+  private void emitStartingTimeMetric(ProgramRunId programRunId, long startDelayTime) {
+    Map<String, String> tags = Collections.emptyMap();
```

Comment:
should we add tags, similar to other metrics emitted here?

Reply:
We can I guess, but won't that be redundant as we can always see those tags in the completion metrics of the same program?